enhancements/machine-api/control-plane-machinesets.md

```diff
@@ -0,0 +1,226 @@
+---
+title: Managing Control Plane machines
+authors:
+  - enxebre
+reviewers:
+  - hexfusion
+  - jeremyeder
+  - abhinavdahiya
+  - joelspeed
+  - smarterclayton
+  - derekwaynecarr
+approvers:
+  - TBD
+
+creation-date: 2020-04-02
+last-updated: yyyy-mm-dd
+status: implementable
+see-also:
+  - https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/proposals/20191017-kubeadm-based-control-plane.md
+  - https://github.com/openshift/enhancements/blob/master/enhancements/etcd/cluster-etcd-operator.md
+  - https://github.com/openshift/enhancements/blob/master/enhancements/etcd/disaster-recovery-with-ceo.md
```

Comment:
Nit: no need for the full URI here, you can be relative to this file or the repo root (see [the template][1]).

[1]: https://github.com/openshift/enhancements/blame/0507a7d644d7e80ce1726f19cc5050be89f99780/enhancements/template.md#L14-L15

Reply:
Actually I can't see how.
Using the template format does not render it as a link, e.g https://github.com/openshift/enhancements/blob/5b705e0b3c303be3f21a175b9483e1ca5ded8f28/enhancements/installer/crc.md

And using link []() syntax seems to completely change the whole header render format, e.g https://github.com/openshift/enhancements/blob/1838b96dab09d466da0653a05006726eab20c187/enhancements/ingress/wildcard-admission-policy.md

Reply:
My impression of this header was that it was being written for some future, hypothetical consumer, and not intended for GitHub's Markdown browser.  But :shrug:, I guess that could go either way, and we can always do some kind of mass edit if/when the consumer is born.